js/interface.js

```diff
@@ -33,6 +33,8 @@ var table;
 var dataSourceEntriesHasChanged = false;
 var isShowingAll = false;
 var columns;
+var dataSourcesToSearch = [];
```

Comment:
Please explain and make a comment on what `dataSourcesToSearch` is for.

Reply:
@squallstar That variable was added in this [PR](https://github.com/Fliplet/fliplet-widget-data-source/pull/68) in order to fix this [issue](https://github.com/Fliplet/fliplet-studio/issues/3841) but there was an error with it because it wasn't declared.